pkg/report/testdata/openbsd/report/8

```diff
@@ -0,0 +1,3 @@
+TITLE: panic: kernel diagnostic assertion "pg->wire_count == 1" failed: file "/sys/kern/vfs_biomem.c", line 329
+
+panic: kernel diagnostic assertion "pg->wire_count == 1" failed: file "/syzkaller/managers/main/kernel/sys/kern/vfs_biomem.c", line 329
```

Comment:
Please add full output.
We used to do just short snippets for linux initially too, but as we improved parsing (extracting functions, detecting corrupted reports, etc), they all become useless (e.g. all detected as corrupted, which they are strictly saying are, one line is not a proper report).
This set of full crash reports, for all different types of crashes, for different kernel versions (maybe not so relevant for openbsd) is extremely valuable asset for testing.